src/common/components/premiumSelect/premiumSelect.component.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular'
+
+import template from './premiumSelect.tpl.html'
+
+const componentName = 'premiumSelect'
+
+class PremiumSelect {
```

Comment:
For the sake of naming convention, this should probably be `PremiumSelectController`